tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -46,7 +46,7 @@
    "outputs": [],
    "source": [
```

Comment:
Throughout the file - I think that "Torch" should be changed to either "torch" (if you want to refer to the package, such as in "torch model" or PyTorch, if you're referring to the framework).
Moreover, you have places where you say "Torch module" and "PyTorch module" in the same paragraph, so just be consistent (e.g., in  a Torch Module)